### PR TITLE
Alibaba: fix: add the availability zone of worker

### PR DESCRIPTION
**Pull request by bd233**
The `ali_zone_ids` needs to contain the availability zone of the workers,
and create the VSwitches

### CODE REVIEW

**Comment by patrickdillon** [comment]
LGTM. I will let @kwoodson take a look
/approve

**Comment by kwoodson** [comment]
I will test this one now. brb

**Review by staebler — COMMENTED** [review]
There is other work that needs to happen first before this can go in as is. Currently, the number of master instances created is equal to the number of vswitches. There is one vswitch for every zone. If we make this change to add workers to the zones, then we will end up creating masters in every zone where we have workers, which is not what the user may want. I think this PR is dependent upon https://github.com/openshift/installer/pull/5379.

The comments for the following terraform variables need to be changed.
https://github.com/openshift/installer/blob/cd69a4bda52fe5e4cf4d250717529ba9e8811ba4/data/data/alibabacloud/variables-alibabacloud.tf#L16
https://github.com/openshift/installer/blob/cd69a4bda52fe5e4cf4d250717529ba9e8811ba4/data/data/alibabacloud/cluster/vpc/variables.tf#L11

**Comment by bd233** [comment]
@staebler  I have update description.

> I think this PR is dependent upon #5379.

That's it.

File: data/data/alibabacloud/variables-alibabacloud.tf [hunk 1]
```diff
@@ -25,7 +25,7 @@ variable "ali_private_zone_id" {
 
 variable "ali_zone_ids" {
   type        = list(string)
-  description = "The availability zones in which to create the masters. The length of this list must match master_count."
+  description = "The availability zones in which to create the masters, workers and bootstrap node."
```
Comment (staebler) [inline data/data/alibabacloud/variables-alibabacloud.tf:28]: This change will break the masters module. The zone ids passed into the master module must be an ordered list corresponding to the masters that need to be created. There needs to be one variable for the master zone ids and a separate variable for the worker zone ids.

**Review by staebler — CHANGES_REQUESTED** [review]

File: data/data/alibabacloud/variables-alibabacloud.tf (hunk 1 above)
Comment (bd233) [reply data/data/alibabacloud/variables-alibabacloud.tf]: Yes, your are right. I have updated.

**Review by staebler — APPROVED** [review]
/lgtm
/approve

**Pull request merged**